pkg/webui/lib/components/require-request.js

```diff
@@ -54,7 +54,7 @@ const RequireRequest = React.memo(
 )
 
 RequireRequest.propTypes = {
-  children: PropTypes.node.isRequired,
+  children: PropTypes.node,
```

Comment:
What was the reason to add remove this? Seems a bit odd.

Reply:
In the status badge component there is the following structure:
```
<RequireRequest requestAction={getNetworkStatusSummary()} handleErrors={false}>
      {status && (
        ...
      )}
</RequireRequest>
```

If the status is undefined, the require request will have no children and therefore there will be a type error in the browser.

Reply:
`{ status ? (…) : null }`  
Should work I believe.